Platform/NXP/Sabre_iMX6SX_1GB/Drivers/GraphicsOutputDxe/GraphicsOutputDxe.c

```diff
@@ -0,0 +1,415 @@
+//// 
```

Comment:
Please fix this header to follow the EDK2 copyright/license style. Applies to all files in this folder.

Reply:
Fixed.